archlinuxcn/waydroid-git/PKGBUILD

```
@@ -29,5 +29,6 @@ pkgver() {
 }
 
 package() {
+	sed -i -s "s/env python/python/g" "${_pkgname}/waydroid.py"
```

```suggestion
	sed -i '1s/env //' "${_pkgname}/waydroid.py"
```
It it unnecessary to use a global substitution.